src/crate/client/sqlalchemy/tests/insert_from_select_test.py

```diff
@@ -64,19 +63,20 @@ class CharacterArchive(Base):
 
         self.character = Character
         self.character_archived = CharacterArchive
-        self.session = Session()
+        self.session = Session(bind=self.engine)
 
     @patch('crate.client.connection.Cursor', FakeCursor)
     def test_insert_from_select_triggered(self):
         char = self.character(name='Arthur', status='Archived')
         self.session.add(char)
         self.session.commit()
 
-        sel = select([self.character.name, self.character.age]).where(self.character.status == "Archived")
+        sel = select(self.character.name, self.character.age).where(self.character.status == "Archived")
         ins = insert(self.character_archived).from_select(['name', 'age'], sel)
         self.session.execute(ins)
         self.session.commit()
+        # TODO: Verify if this is correct.
         self.assertSQL(
-            "INSERT INTO characters_archive (name, age) SELECT characters.name, characters.age FROM characters WHERE characters.status = ?",
+            "INSERT INTO characters_archive (name, age) SELECT characters.name, characters.age FROM characters WHERE characters.status = :status_1",
```

Comment:
I believe this change is legitimate, and is coming from the new [Transparent SQL Compilation Caching](https://docs.sqlalchemy.org/en/20/changelog/migration_14.html#transparent-sql-compilation-caching-added-to-all-dql-dml-statements-in-core-orm) feature added with SQLAlchemy 1.4.

However, I still would like to validate the functionality beyond this test, to make sure my assumption is correct.

Reply:
I would like to contradict my statement. 00384515 brings back the "qmark" paramstyle used by the CrateDB Python driver. For more details, see GH-490.